lib/std/io/tty.zig

```diff
@@ -0,0 +1,155 @@
+                        .white => "\x1b[37m",
+                        .bright_black => "\x1b[90m",
+                        .bright_red => "\x1b[91m",
+                        .bright_green => "\x1b[92m",
+                        .bright_yellow => "\x1b[93m",
+                        .bright_blue => "\x1b[94m",
+                        .bright_magenta => "\x1b[95m",
+                        .bright_cyan => "\x1b[96m",
+                        .bright_white => "\x1b[97m",
+                        .bold => "\x1b[1m",
+                        .dim => "\x1b[2m",
+                        .reset => "\x1b[0m",
+                    };
+                    try out_stream.writeAll(color_string);
+                },
+                .windows_api => |ctx| if (native_os == .windows) {
+                    const attributes = switch (color) {
+                        .black => 0,
+                        .red => windows.FOREGROUND_RED,
+                        .green => windows.FOREGROUND_GREEN,
+                        .yellow => windows.FOREGROUND_RED | windows.FOREGROUND_GREEN,
+                        .blue => windows.FOREGROUND_BLUE,
+                        .magenta => windows.FOREGROUND_RED | windows.FOREGROUND_BLUE,
+                        .cyan => windows.FOREGROUND_GREEN | windows.FOREGROUND_BLUE,
+                        .white => windows.FOREGROUND_RED | windows.FOREGROUND_GREEN | windows.FOREGROUND_BLUE,
+                        .bright_black => windows.FOREGROUND_INTENSITY,
+                        .bright_red => windows.FOREGROUND_RED | windows.FOREGROUND_INTENSITY,
+                        .bright_green => windows.FOREGROUND_GREEN | windows.FOREGROUND_INTENSITY,
+                        .bright_yellow => windows.FOREGROUND_RED | windows.FOREGROUND_GREEN | windows.FOREGROUND_INTENSITY,
+                        .bright_blue => windows.FOREGROUND_BLUE | windows.FOREGROUND_INTENSITY,
+                        .bright_magenta => windows.FOREGROUND_RED | windows.FOREGROUND_BLUE | windows.FOREGROUND_INTENSITY,
+                        .bright_cyan => windows.FOREGROUND_GREEN | windows.FOREGROUND_BLUE | windows.FOREGROUND_INTENSITY,
+                        .bright_white, .bold => windows.FOREGROUND_RED | windows.FOREGROUND_GREEN | windows.FOREGROUND_BLUE | windows.FOREGROUND_INTENSITY,
+                        // "dim" is not supported using basic character attributes, but let's still make it do *something*.
+                        // This matches the old behavior of TTY.Color before the bright variants were added.
+                        .dim => windows.FOREGROUND_INTENSITY,
+                        .reset => ctx.reset_attributes,
+                    };
+                    try windows.SetConsoleTextAttribute(ctx.handle, attributes);
+                } else {
+                    unreachable;
+                },
+            };
+        }
+
+        pub fn writeDEC(conf: Config, writer: anytype, codepoint: u8) !void {
```

Comment:
I still don't really understand this one. Last time I looked this is not used anywhere in the compiler and I think when @andrewrk added this in the std.Build PR he forgot to make use of it? The build summary drawing code does not use this.

Reply:
Me neither, so unless someone tells me to remove it I will just leave it there :^)

Reply:
I used it briefly until Mitchell informed me that there were terminal escape codes that could be used to directly create the shapes without switching the "mode" of the terminal. So I used those instead, and then forgot to remove this writeDEC function. Feel free to remove it.

Reply:
Thanks, done :+1: